### ISSUE TITLE
Support pvc size expansion

### ISSUE THREAD

**Issue by sepich** [2gis/k8s-handle#124]
Hello,
Currently PVC size increase is not supported via k8s-handle:
```
[10:39:09]W:	 [Step 4/4] 2020-04-03 07:39:09 INFO:k8s_handle.k8s.provisioner:PersistentVolumeClaim "thanos-compact" already exists, replace it
[10:39:09]W:	 [Step 4/4] 2020-04-03 07:39:09 ERROR:k8s_handle.k8s.provisioner:{'storage': '100Gi'} != {'storage': '200Gi'}
[10:39:09]W:	 [Step 4/4] 2020-04-03 07:39:09 ERROR:k8s_handle.k8s.adapters:Exception when calling "replace_namespaced_persistent_volume_claim": {
[10:39:09]W:	 [Step 4/4]     "kind": "Status",
[10:39:09]W:	 [Step 4/4]     "apiVersion": "v1",
[10:39:09]W:	 [Step 4/4]     "metadata": {},
[10:39:09]W:	 [Step 4/4]     "status": "Failure",
[10:39:09]W:	 [Step 4/4]     "message": "PersistentVolumeClaim \"thanos-compact\" is invalid: spec: Forbidden: is immutable after creation except resources.requests for bound claims",
[10:39:09]W:	 [Step 4/4]     "reason": "Invalid",
[10:39:09]W:	 [Step 4/4]     "details": {
[10:39:09]W:	 [Step 4/4]         "name": "thanos-compact",
[10:39:09]W:	 [Step 4/4]         "kind": "PersistentVolumeClaim",
[10:39:09]W:	 [Step 4/4]         "causes": [
[10:39:09]W:	 [Step 4/4]             {
[10:39:09]W:	 [Step 4/4]                 "reason": "FieldValueForbidden",
[10:39:09]W:	 [Step 4/4]                 "message": "Forbidden: is immutable after creation except resources.requests for bound claims",
[10:39:09]W:	 [Step 4/4]                 "field": "spec"
[10:39:09]W:	 [Step 4/4]             }
[10:39:09]W:	 [Step 4/4]         ]
[10:39:09]W:	 [Step 4/4]     },
[10:39:09]W:	 [Step 4/4]     "code": 422
[10:39:09]W:	 [Step 4/4] }
  Process exited with code 1
```
But I can edit it manually, its not immutable:
```
$ k get pvc thanos-compact
NAME                                   STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
thanos-compact                         Bound    pvc-d060c6b0-f1c5-4423-bda3-88eb09678421   100Gi      RWO            default        3d20h
$ k edit pvc thanos-compact
persistentvolumeclaim/thanos-compact edited
$ k get pvc thanos-compact
NAME             STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
thanos-compact   Bound    pvc-d060c6b0-f1c5-4423-bda3-88eb09678421   200Gi      RWO            default        3d20h
```
And after this k8s-handle pass:
```
  2020-04-03 07:44:02 INFO:k8s_handle.k8s.provisioner:PersistentVolumeClaim "thanos-compact" already exists, replace it
  2020-04-03 07:44:02 INFO:k8s_handle.k8s.provisioner:PersistentVolumeClaim is not changed
```

**Comment by dekhtyarev** [comment]
Hi, @sepich !
Thank you for checking this case! As soon as we have the resources we will definitely fix it.

**Comment by rvadim** [comment]
@sepich @dekhtyarev fix is ready, but im need to merge #122 first.
